airflow/providers/google/common/hooks/base_google.py

```diff
@@ -454,6 +474,24 @@ def decorator(fun: T):
 
         return decorator
 
+    @staticmethod
+    def refresh_credentials_retry(*args, **kwargs) -> Callable[[T], T]:
+        """Provide a mechanism to repeat requests in response to a temporary refresh credential issue."""
+
+        def decorator(fun: T):
```

Comment:
nitpick: `func` seems to be a more commonly used convention in airflow 

```suggestion
        def decorator(func: T):
```

Reply:
Changed. Changed also for all other decorator in `base_google.py`

Reply:
Nice. Thanks for the change!